DOCUMENTATION.md

```diff
@@ -128,9 +127,9 @@ If you feel something is missing, not clear or could be improved, please don't h
         * [.whoami()](#resin.auth.whoami) ⇒ <code>Promise</code>
         * [.authenticate(credentials)](#resin.auth.authenticate) ⇒ <code>Promise</code>
         * [.login(credentials)](#resin.auth.login) ⇒ <code>Promise</code>
-        * [.loginWithToken(token)](#resin.auth.loginWithToken) ⇒ <code>Promise</code>
+        * [.loginWithToken(authToken)](#resin.auth.loginWithToken) ⇒ <code>Promise</code>
         * [.isLoggedIn()](#resin.auth.isLoggedIn) ⇒ <code>Promise</code>
-        * [.getToken()](#resin.auth.getToken) ⇒ <code>Promise</code>
+        * [.getKey()](#resin.auth.getKey) ⇒ <code>Promise</code>
```

Comment:
Sorry, I know I've flip-flopped on this a bit, but I think this should stay as `.getToken`.

Before I wanted to change this and `loginWithToken` to `getKey`/`loginWithKey`, but Ariel pointed out that's the wrong terminology on the previous PR (https://github.com/resin-io/resin-sdk/pull/439#issuecomment-347612074). We should keep the two consistent I think.